hub_mozita.py

```diff
@@ -58,7 +58,6 @@ def safe_conf_get(config_parser, section, key_name):
 ###########################
 #  MANAGING BOT CONSTANTS #
 ###########################
-TOKEN = safe_conf_get(config_parser, "bot", "TOKEN")
```

Comment:
Era un duplicato di https://github.com/MozillaItalia/mozitahub_bot/blob/master/hub_mozita.py?rgh-link-date=2020-10-06T07%3A43%3A21Z#L41 che tra l'altro sovrascriveva

Reply:
Sfondi tutto così, non c'è più il token, non viene salvato

Reply:
Esplicita dove sarebbero le linee duplicate @mte90

Reply:
@Mte90 @dag7dev scusate se intervengo. Non sto capendo quale è il problema.
Io non trovo righe duplicate e questa riga dovrebbe servire per ottenere il TOKEN dal file, altrimenti non esiste

Reply:
Allora prendete il codice attuale, https://github.com/MozillaItalia/mozitahub_bot/blob/master/hub_mozita.py#L41 come vedete leggiamo un token e lo assegniamo alla variabile token.
Poi questa PR fa la stessa cosa solo che ha cambiato i parametri nel file config.ini senza comunicarlo ma lasciando il file config-sample.ini con il vecchio parametro di configurazione.
Nella mia revisione quindi qualcosa non tornava, mi sono accorto ora che la linea non era duplicata ma i parametri del file config sono stati cambiati.  
Ripristino la riga ma il file config va aggiornato per rispecchiare la configurazione.

Reply:
@Mte90 Io vedo anche un nuovo file "config-sample.ini" in realtà :thinking: 

In ogni caso, tutto risolto ora :smile: